sympy/physics/control/lti.py

```diff
@@ -422,6 +423,43 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
     return wrapper
 
+def routh_table(system, first_col=False):
```

Comment:
```suggestion
def routh_hurwitz_table(...)
```

Reply:
To be generic it should form the table simply from a polynomial expression.

Reply:
Should I remove the TransferFunction condition from the code ?